lua/tardis/settings/cl_performance.lua

```diff
@@ -13,15 +13,27 @@ TARDIS:AddSetting({
 })
 
 TARDIS:AddSetting({
-    id="projlights-enabled",
+    id="extra-lights-disabled",
```

Comment:
Can we inverse this option? Currently all the performance options require unticking to turn features off so this one is the odd one out 

![image](https://user-images.githubusercontent.com/2363642/204428883-3d60331d-fc22-4645-8bd6-29e116c3ad4f.png)

Reply:
Fixed